doc/en/source/ngsildbasediotintegration.rst

```diff
@@ -0,0 +1,476 @@
+
+.. figure:: figures/dashboard.png
+
+
+
+How to Fetch data from IoT Agent to Orion-LD
+================================================================
+
+**Step1** Provisioning a Service Group
+-----------------------------------------------------------------
+Invoking group provision is always the first step in connecting devices since it is always necessary to supply an authentication key with each measurement and the IoT Agent will not initially know which URL the context broker is responding on.
+
+It is also possible to set up default commands and attributes for all anonymous devices as well, but this is not done within this tutorial as we will be provisioning each device separately.
+
+This example provisions an anonymous group of devices. It tells the IoT Agent that a series of devices will be sending messages to the IOTA_HTTP_PORT (where the IoT Agent is listening for Northbound communications)
+
+.. code-block:: console   
+
+	curl -iX POST 'http://localhost:4041/iot/services' \
+	-H 'fiware-service: openiot' \
+	-H 'fiware-servicepath: /' \
+	-H 'Content-Type: application/json' \
+	--data-raw '{
+    	"services": [
+        	{
+            	"apikey": "4jggokgpepnvsb2uv4s40d59ov",
+            	"cbroker": "http://orion:1026",
+            	"entity_type": "Device",
+            	"resource": "/iot/d",
+            	"attributes": [
+                	{
+                    	"object_id": "bpm", "type": "Property", "name": "heartRate",
+                    	"metadata": { "unitCode": {"type": "Text", "value": "5K" }}
+                	},
+                	{
+                    	"object_id": "s", "name": "status", "type": "Property"
+                	},
+                	{
+                    	"object_id": "gps", "name": "location", "type": "geo:point"
+                	}
+            	],
+            	"static_attributes": [
+                	{
+                    	"name": "category", "type": "Property", "value": "sensor"
+                	},
+                	{
+                    "name": "supportedProtocol", "type": "Property", "value": "ul20"
+                	}
+            	]
+        	}
+    	   ]
+	}'
+
+
+**Note:** To know about Fiware-Service and Fiware-ServicePath please click  `here`_
+
+.. _`here`: https://ngsi-ld-tutorials.readthedocs.io/en/latest/iot-agent.html#connecting-iot-devices
+
+
+In the example the IoT Agent is informed that the `/iot/d` endpoint will be used and that devices will authenticate
+themselves by including the token **4jggokgpepnvsb2uv4s40d59ov**. For an UltraLight IoT Agent this means devices will be
+sending GET or POST requests to:**http://iot-agent:7896/iot/d?i=<device_id>&k=4jggokgpepnvsb2uv4s40d59ov**
+
+**Step2** Provisioning an Actuator
+----------------------------------
+The example below provisions a water with the `deviceId=water001`. The endpoint is
+`http://iot-sensors:3001/iot/water001` and it can accept the `on` command. The `transport=HTTP` attribute defines the
```

Comment:
Better to write 'on' in capital letter or surrounded by inverted commas.

Reply:
Done